packages/ui/components/scheduler/_storybookData.ts

```diff
@@ -0,0 +1,71 @@
+import dayjs from "@calcom/dayjs";
```

Comment:
Storybook data used in stories.

Sinde we use mdx files - its a lot easier defining this here than inside the file itself